### PR TITLE
Prepare milestone 3

### PR DESCRIPTION
**Pull request by blackinno**
- [x] update readme with step by step commands to install the app on a DAO
- [x] add useful commands for other developers (list version, publish, etc...)
- [x] create a new demo video that shows the new UI 
- [x] add/update description to README of what the app can do (list services/tasks) and what needs to be done (add deletion of connections)
- [x] remove Aragon app description and screenshots https://github.com/mesg-foundation/aragon/blob/master/app/public/meta/
- [x] publish it on Aragon rinkeby, make sure to update the commands if needed

Link of the video: https://drive.google.com/open?id=1juRUiNeuWwtDjrBT5e4zxNJR45vKvkf4

Step of the video: https://docs.google.com/document/d/1XwhotkhraBFhRuVX1a6Fm9Gbke6Eskob7PpwRlRSe68/edit?usp=sharing

View Link: https://rinkeby.aragon.org/#/0x702E819000FC3Be0dadC99480401bD94DBD5b322

Close https://github.com/mesg-foundation/aragon/issues/29.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,106 @@
...
+
+- `Deactivate`, It's a feature to remove connection from an event and don't send notification.
+- Add _**zapier service**_.
+- Add _**twilio service**_.
+
 ## Demo
 
 Check out the demo video:
 
 [![Demo: Connect your DAO events to any webhook](https://img.youtube.com/vi/MctlHrLSm70/0.jpg)](https://www.youtube.com/watch?v=MctlHrLSm70 "Demo: Connect your DAO events to any webhook")
```
Comment (NicolasMahe) [inline README.md:24]: Let's update the video with the new one.
@williamzmorgan could you publish the new one to youtube and replace the link here

Comment (williamzmorgan) [reply README.md]: @NicolasMahe I've updated the video link. 👍

Comment (NicolasMahe) [reply README.md]: @williamzmorgan thanks but you forget to update the preview image.

Comment (williamzmorgan) [reply README.md]: Ok, I've added a URL to the new video thumbnail, and also updated the link to the new video that is not blocking the Telegram message.

**Pull request merged**